### PR TITLE
Imgix dev

### PR DESCRIPTION
**Pull request by syedbadshah-toptal**

### CODE REVIEW

File: cartridges/int_imgix_pd/cartridge/experience/components/imgix/imageComponent.js [hunk 1]
```diff
@@ -1,21 +1,20 @@
...
-  model.title = content.title ? content.title : null;
-  model.image = ImageTransformation.getScaledImage(content.image);
-  // use to give link on the image, if we click on image it take to us to that page.
-  model.link = content.imageLink ? content.imageLink : "#";
-  model.alt = content.alt ? content.alt : null;
+/**
+ * Render logic for storefront.productBannerStrip component.
+ * @param {dw.experience.ComponentScriptContext} context The Component script context object.
+ * @returns {string} The template to be displayed
+ */
```
Comment (frederickfogerty) [inline cartridges/int_imgix_pd/cartridge/experience/components/imgix/imageComponent.js:11]: Is this comment accurate?

File: cartridges/int_imgix_pd/cartridge/experience/components/imgix/imageComponent.json [hunk 2]
```diff
@@ -1,24 +1,33 @@
...
           "editor_definition": {
-            "type": "imgix.imageSelection"
+            "type": "imgix.imgixEditorTrigger",
+            "configuration": {
+              "options": {
+                "config": [
+                  "Yugo",
+                  "Golf",
+                  "Mercedes"
+                ]
```
Comment (frederickfogerty) [inline cartridges/int_imgix_pd/cartridge/experience/components/imgix/imageComponent.json:22]: These configs should probably be removed

**Review by frederickfogerty — APPROVED** [review]
LGTM, once the comments below have been addressed

**Review by luqven — APPROVED** [review]
LGTM except the [package-lock.json file](https://github.com/imgix/sf-commerce-cloud/blob/97c8dbf5e8b81814525d5eb7f1033dae15bc6b4f/package-lock.json) shouldn't be committed. This repo already has a `yarn.lock`.

Let's remove that file before merging.

**Review by luqven — APPROVED** [review]
LGTM if Fred's feedback above has been resolved ☝🏼

File: cartridges/int_imgix_pd/cartridge/experience/components/imgix/imageComponent.js [hunk 3]
```diff
@@ -1,21 +1,20 @@
...
 
-  return new Template("/imgix/image/imagerenderer/imagerenderer").render(model)
-    .text;
+module.exports.render = function (context) {
```
Comment (frederickfogerty) [inline cartridges/int_imgix_pd/cartridge/experience/components/imgix/imageComponent.js:13]: Now that js-core is integrated into the `next` branch, it's important those changes aren't overwritten. Therefore this needs to integrate those changes.

**Review by frederickfogerty — CHANGES_REQUESTED** [review]

File: cartridges/bm_imgix_pd/cartridge/experience/editors/imgix/imgixEditorTriggerBreakout.json [hunk 4]
```diff
@@ -0,0 +1,14 @@
+{
+  "name": "Custom Car Editor Breakout",
```
Comment (frederickfogerty) [inline cartridges/bm_imgix_pd/cartridge/experience/editors/imgix/imgixEditorTriggerBreakout.json:2]: This should be updated

**Comment by luqven** [comment]
@frederickfogerty @syedbadshah-toptal I think we should close this in favor of #66 to avoid overwriting some of Fred's work in #64

**Comment by frederickfogerty** [comment]
I agree with this @luqven 👍  @syedbadshah-toptal what do you think?

**Pull request closed without merging**